ftl/core/deck-config.ftl

```diff
@@ -448,7 +448,10 @@ deck-config-answer-hard = Answer Hard
 deck-config-answer-good = Answer Good
 deck-config-days-to-simulate = Days to simulate
 deck-config-desired-retention-below-optimal = Your desired retention is below optimal. Increasing it is recommended.
-
+deck-config-question-action-show-answer = Show Answer
+deck-config-question-action-show-reminder = Show Reminder
+deck-config-question-action = Question action 
+deck-config-question-action-tool-tip = The action to perform after the end of the time of the question. Either show answer and auto update or just show a reminder.
```

Comment:
Let's drop the second sentence, as it simply repeats the dropdown options. Suggested wording:

The action to perform after the question is shown, and time has elapsed.